packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2908,6 +2908,67 @@ describe("Runtime", () => {
 				);
 			});
 
+			it("ignores signals sent while disconnected and resets stats on reconnect", () => {
+				// SETUP - define resubmit and setConnectionState on channel collection.
+				// This is needed to submit test data store ops. Once defined, submit a test data store op
+				// so that message is queued in PendingStateManager and reconnect count is increased.
+				// Send and process an initial signal to prime the system.
+				// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+				(containerRuntime as any).channelCollection = {
+					setConnectionState: (_connected: boolean, _clientId?: string) => {},
+					// Pass data store op right back to ContainerRuntime
+					reSubmit: (type: string, envelope: any, localOpMetadata: unknown) => {
+						submitDataStoreOp(
+							containerRuntime,
+							envelope.address,
+							envelope.contents,
+							localOpMetadata,
+						);
+					},
+				} as ChannelCollection;
+				submitDataStoreOp(containerRuntime, "1", "test");
+				sendSignals(1);
```

Comment:
Let's move the `// Send and process an initial signal to prime the system.` comment to just above where that happens (above `submitDataStoreOp`)